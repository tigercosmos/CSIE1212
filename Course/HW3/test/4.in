20
O...O
..O..
OOOXX
..XXX
O.X.O

X.OOX
.XX..
.OOO.
.OOO.
X...X

X...X
X.O.X
.OOOO
OOO..
X...X

X...X
.XXOO
.OO..
.OOOO
X...X

XOXO.
.XOX.
.OOOO
.XOX.
.....

OXOXO
.XOX.
.OO..
.XOX.
.O...

OXOXX
.XO..
..O.O
.XOX.
O..O.

XO..X
.OXO.
X.O.O
.XOX.
O..O.

O.O.X
.XO..
O.X.O
XXOX.
O..O.

XOXXO
O...O
O...X
....O
XOOX.

O...O
..O..
OO.XX
O.XXX
O.X.O

XOOOX
.XX..
.O.O.
.OOO.
X...X

XO..X
X.O.X
.OO.O
OOO..
X...X

X...X
.XXOO
.OO..
.OOOO
..X.X

.OXO.
.XOX.
.OOOO
.XOX.
X....

OXOXO
..OX.
.OO..
.XOX.
.O..X

OX.XX
.XO..
..O.O
OXOX.
O..O.

XO..X
.OXO.
X...O
.XOX.
O..OO

O.O.X
OX...
O.X.O
XXOX.
O..O.

XOXXO
O...O
O.O.X
....O
X.OX.