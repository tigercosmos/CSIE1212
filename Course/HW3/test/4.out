O win
Draw
X win
Draw
X win
Draw
Draw
Draw
X win
Draw
O win
X win
X win
Draw
X win
Draw
Draw
Draw
X win
Draw
